src/scalper.ts

```diff
@@ -732,69 +705,65 @@ Spot Δ: ${spotDelta} Offset Δ ${this.deltaOffset} Fair Value: ${fairValue}`,
       console.log(this.symbol, 'Jupiter Route', err, err.stack);
     }
 
-    try {
```

Comment:
Order failures will pop up and just run the gamma scalper now? Did you test at all?

Reply:
Dont know of a good way to test forcing failures on these RPCs, so didnt.
Will eventually get to writing tests, but those are hard on code with so many external dependencies

Reply:
A big reason for all the effort to split up the code and modularize it is to be able to test it, or at least most of it